api/trello/api.go

```diff
@@ -0,0 +1,83 @@
+package trello
+
+import(
+  "net/url"
+  "net/http"
+  "encoding/json"
+  "io/ioutil"
+)
+
+const API_KEY = "[API_KEY]"
+
+var LinkForAuthKey = "https://trello.com/1/authorize?expiration=never&scope=read,write&response_type=token&name=Treit&key=" + API_KEY
+
+func CreateNewBoard(authKey string, boardname string) (board Board) {
+  response, err := http.Post("https://api.trello.com/1/boards?name=" + boardname + "&key=" + API_KEY + "&token=" + authKey, "", nil)
+  if err != nil {
+    // TODO: Handle error
+  }
+  defer response.Body.Close()
+  body, err := ioutil.ReadAll(response.Body)
+  if err != nil {
+    // TODO: Handle error
+  }
+  board = Board{}
+  json.Unmarshal(body, &board)
+  return
```

Comment:
Does this actually work ? This should not compile

Reply:
Why shouldn't it work? 🤔 
This feature is [named return values](https://tour.golang.org/basics/7)..

Reply:
ah.. got it.. You are right. I just never used this feature.